.golangci.yaml

```diff
@@ -20,14 +21,11 @@ linters:
   - asciicheck
   - containedctx
   - decorder
-  - depguard
```

Comment:
This was never configured, and it defaults to stdlib only, which is not useful in this case

Reply:
:+1:  We can always bring it back when we need it.